src/main/java/frc/robot/util/ShuffleBoard/Tabs/AutoTab.java

```diff
@@ -44,14 +44,14 @@ public AutoTab(Drivetrain drive, Shooter shooter, Arm arm, StorageIndex indexer,
     
     public void createEntries(){  
         tab = Shuffleboard.getTab("Auto");
-        autoCommand.setDefaultOption("Do nothing", new DoNothing());
-        autoCommand.addOption("5 piece (original-tested)", new FollowPathCommand("5 piece (original-tested)",true, drive));
-        autoCommand.addOption("5.5 piece (original-tested)", new FollowPathCommand("5.5 piece (original-tested)",true, drive));
-        autoCommand.addOption("3 piece [v3]", new FollowPathCommand("3 piece [v3]",true, drive));
-        autoCommand.addOption("6 piece (new)", new FollowPathCommand("6 piece (new)",true, drive));
-        autoCommand.addOption("5 piece (original-tested) [V2]", new FollowPathCommand("5 piece (original-tested) [V2]",true, drive));
-        autoCommand.addOption("3.5 piece [V2]", new FollowPathCommand("3.5 piece [V2]",true, drive));
-        autoCommand.addOption("1 piece", new SequentialCommandGroup(prepare(),new WaitCommand(3),index(),new WaitCommand(2),new PrepareShooter(shooter, 0)));
+        // autoCommand.setDefaultOption("Do nothing", new DoNothing());
+        // autoCommand.addOption("5 piece (original-tested)", new FollowPathCommand("5 piece (original-tested)",true, drive));
```

Comment:
Are we intending to remove the options for auto commands on all robots (including Vivace)? This looks like it will prevent any auto command options from showing on Shuffleboard. Just want to make sure I understand properly.

Reply:
Yes. Defining these causes problems when running on Phil and causes the code to crash. Although we could add if statements instead, unless we plan to run our autos from last year (without a speaker), we don't need these.